package.json

```
@@ -14,7 +14,10 @@
     "watch:tokens": "yarn build && chokidar \"tokens/**/*.json\" -c \"yarn build\"",
     "semantic-release": "semantic-release",
     "prepublishOnly": "npm run build",
-    "commit": "cz"
+    "commit": "cz",
+    "test:unit": "vitest",
+    "test:unit:open": "vitest --ui",
+    "test:unit:coverage": "vitest run --coverage"
```

nit: since we won't have another type test here, let's update the commands to just `test`
```suggestion
    "test": "vitest",
    "test:open": "vitest --ui",
    "test:coverage": "vitest run --coverage"
```